<launch>

    <include file="/home/mekatronom/boschmekatronom_ws/BoschMekatronom/src/zed-ros-wrapper/zed_wrapper/launch/zed2i.launch"/>

    <node  pkg ="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan">
        <remap from="cloud_in" to="/zed2i/zed_node/point_cloud/cloud_registered"/>
        <remap from="scan" to="/zed2i/depth/scan"/>
        <rosparam>
            target_frame: "base_link"
            transform_tolerance: 0.01

            min_height: -0.12
            max_height: 0.4
            
            angle_min: -0.54
            angle_max: 0.54
            angle_increment: 0.0087
            scan_time: 0.0333
            range_min: 0.3
            range_max: 2.5
            use_inf: true
            concurrency_level: 1
        </rosparam>
    </node>

</launch>